### PR TITLE
Support starting and stopping runners in batch 

### PR DESCRIPTION
**Pull request by variant77**

### CODE REVIEW

**Review by GreasyAvocado — APPROVED** [review]

File: action.yml [hunk 1]
```diff
@@ -72,6 +85,10 @@ outputs:
     description: >-
       EC2 Instance Id of the created runner.
       The id is used to terminate the EC2 instance when the runner is not needed anymore.
+  instancesDetail:
+    description: >-
+      EC2 Instance Id and label for each created runner.
```
Comment (itai-viz) [inline action.yml:90]: And name, right?

File: src/index.js [hunk 2]
```diff
@@ -7,24 +7,70 @@ function setOutput(label, ec2InstanceId) {
...
 
-async function start() {
+async function startInstance(name) {
   const label = config.generateUniqueLabel();
   const githubRegistrationToken = await gh.getRegistrationToken();
   const ec2InstanceId = await aws.startEc2Instance(label, githubRegistrationToken);
-  setOutput(label, ec2InstanceId);
   await aws.waitForInstanceRunning(ec2InstanceId);
   await gh.waitForRunnerRegistered(label);
+  return {label, ec2InstanceId, name};
```
Comment (itai-viz) [inline src/index.js:20]: What is this returning? A list? I don't understand where the dictionary aspect comes into play. Before it explicitly set `label` to `label` and `ec2-instance-id` to `ec2InstanceId`

File: action.yml (hunk 1 above)
Comment (variant77) [reply action.yml:90]: Technically yes although it is not used in order to stop the instances so I don't care if it's in there or not

File: src/index.js (hunk 2 above)
Comment (variant77) [reply src/index.js:20]: This is a javascript syntax for creating the object {"label": label, "ec2InstanceId": ec2InstanceId, "name": name}

**Pull request merged**